src/test/java/org/jabref/logic/remote/RemoteCommunicationTest.java

```diff
@@ -51,7 +51,7 @@ void pingReturnsTrue() throws IOException, InterruptedException {
     void commandLineArgumentSinglePassedToServer() {
         final String[] message = new String[]{"my message"};
 
-        client.sendCommandLineArguments(message);
+        client.sendCommandLineArguments(message.clone());
```

Comment:
I would move the clone call to the sendMessage method. It's usually a good practice to not modify parameters passed to a method.

Reply:
Done. Thank you for your advice!